app-emulation/euae/euae-0.8.29~wip4.recipe

```diff
@@ -44,6 +44,7 @@ REQUIRES="
 	lib:libGL
 	lib:libglu
 	lib:libpng
+	lib:libschily
```

Comment:
This should be removed, we don't have lib:libschily :)

Reply:
Fixed recipe.